src/main/webapp/view.js

```diff
@@ -15,22 +15,53 @@
 const PARAM_ID = 'id';
 const PARAM_VIDEO_ID = 'video-id';
 
+/** Timestamp that video, disccussion, and transcript were synced to. */
+let lastSyncedTime;
+
 /* exported LECTURE_ID */
 window.LECTURE_ID = getLectureId();
 window.VIDEO_ID = getVideoId();
 
 initialize();
 
 /**
- * Initializes the video player and discussion section
+ * Initializes the timer, video player, and discussion section
  * for the lecture view page.
  */
 async function initialize() {
   window.loadVideoApi();
-  window.intializeDiscussion();
+  window.loadDiscussion();
+  // TODO: Adjust time interval.
+  window.timer = window.setInterval(syncEverythingToVideo, 1000);
   // TODO: Initialize the trancript section.
 }
 
+/**
+ * Retrives the current time of the video and relays time to
+ * {@code sync}.
+ */
+function syncEverythingToVideo() {
+  const currentTime = window.videoPlayer.getCurrentTime();
+  sync(currentTime, false);
+}
+
+/**
+ * Calls functions that seek video, transcript,
+ * and discussion to {@code currentTime}.
+ */
+function sync(currentTime, syncVideo = false) {
```

Comment:
Oops, i'm sorry.  I think this should actually default to true, so discussion and transcript don't need to specify it...